Src/Base/AMReX_GpuAtomic.H

```diff
@@ -232,12 +232,12 @@ namespace detail {
 
     template<class T>
     AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
-    T Max (T* m, T value) noexcept
+    T Max (T* const m, T value) noexcept
```

Comment:
T value -> T const value for consistence?

Reply:
thank you, that slipped through!